### PR TITLE
Revert "Remove InteractiveWindow from Willow authoring (#13588)"

### PR DESCRIPTION
**Pull request by tmat**
This reverts commit b9dcaa13cba7344a8cc49f56256005eae5c77edd.

### CODE REVIEW

**Comment by tmat** [comment]
@KevinH-MS @ManishJayaswal One more change is needed to be reverted directly in lab/ml.

File: src/Setup/DevDivVsix/MicrosoftCodeAnalysisLanguageServices/Microsoft.CodeAnalysis.LanguageServices.vsmanproj [hunk 1]
```diff
@@ -13,6 +13,10 @@
   <Import Project="$(NuGetPackageRoot)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.props" />
   <Import Project="$(NuGetPackageRoot)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.targets" />
 
+  <ItemGroup>
+    <ProjectReference Include="..\MicrosoftVisualStudioInteractiveWindow\Microsoft.VisualStudio.InteractiveWindow.vsmanproj" />
+  </ItemGroup>
+
```
Comment (KevinH-MS) [inline src/Setup/DevDivVsix/MicrosoftCodeAnalysisLanguageServices/Microsoft.CodeAnalysis.LanguageServices.vsmanproj]: Just for my own edification, why is this reference in LanguageServices (instead o some interactive specific project)?

**Comment by KevinH-MS** [comment]
I don't understand what these do, but I see no reason not to ```revert``` 👍

File: src/Setup/DevDivVsix/MicrosoftCodeAnalysisLanguageServices/Microsoft.CodeAnalysis.LanguageServices.vsmanproj (hunk 1 above)
Comment (tmat) [inline src/Setup/DevDivVsix/MicrosoftCodeAnalysisLanguageServices/Microsoft.CodeAnalysis.LanguageServices.vsmanproj]: Microsoft.CodeAnalysis.LanguageServices.vsmanproj is an umbrella for all Roslyn stuff, including interactive language services.

Comment (KevinH-MS) [inline src/Setup/DevDivVsix/MicrosoftCodeAnalysisLanguageServices/Microsoft.CodeAnalysis.LanguageServices.vsmanproj]: I see 👍

**Comment by tmat** [comment]
I guess this should target dev15-preview-5 branch.

**Pull request closed without merging**